contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6057,6 +6058,7 @@ static List *
 transformSelectIntoStmt(CreateTableAsStmt *stmt)
 {
 	List *result;
+	List *babel_4926_original_name_list;
```

Comment:
Let's not use jira numbers in the actual code, use some other generic name for the variables.